tests/main/lxd-mount-units/task.yaml

```diff
@@ -0,0 +1,50 @@
+summary: Test mount units generated for snaps inside lxd container are correct.
+
+details: |
+    Test that mount units generated by snapd-generator inside lxd container
+    are correct for snapfuse.
+
+# only 20.10+, we want lxd images that come with snaps preinstalled.
+systems: [ubuntu-20.10*]
```

Comment:
perhaps systems: [ubuntu-2*, -ubuntu-20.04] ?

Reply:
It actually can be enabled on 20.04, I didn't realize 20.04 lxc images come with core18 preinstalled. Both are enabled now.